spec/test-data/translations/ios_generate_strings_file_from_code/sample-project/Sources/Doc.md

```diff
@@ -0,0 +1,14 @@
+This app will be translated via our `ios_generate_strings_file_from_code` fastlane action which will generate the appropriate `.strings` files.
+
+In particular, the action will detect the following in your Swift code:
+
+```swift
+    NSLocalizedString("key", comment: "translator context")
+    NSLocalizedString("key", value: "English copy", comment: "translator context")
+    NSLocalizedString("key", bundle: Bundle(for: BundleToken.self), value: "English copy", comment: "translator context")
+    NSLocalizedString("key", tableName: "FileName", value: "English copy", comment: "translator context")
+```
+
+And also similar calls in your Objective-C code, and `Text(…)` calls in SwiftUI.
+
+Note that it should not detect the instances of those in `.md` files like this one though.
```

Comment:
I'm pretty sure this is in `sample-project/Sources` rather than `sample-project/` because the tests uses the `Sources` folder. Correct?

Did you consider calling this `README.md`?

Reply:
Yes, I've added 2 different `.md` files in the test data:
 - One under `Pods/SomePod/README.md`
 - This one under `Sources/Doc.md`

The idea was to ensure that none of those 2 `md` files were scanned by `genstrings` (and incorrectly consider the `NSLocalizedString` text in them to be valid keys to add to the generated `.strings` file) in both of the with/without Pods cases. I could have only added one of the 2 and consider that if it's not parsed in the "with Pods" case, then there was no reason why the md would be parsed in the "without Pods" either, but having 2 files to cover both cases didn't hurt 🤷 

As for naming this `README.md`, since I already named the one in `Pods/SomePod/` as `README.md`, I wanted to use a different name here on purpose in order to ensure that any exclusion we'd do in our logic to ensure we only scan `.m` and `.swift` files would not just exclude the `README.md` common name but any `*.md` file whatever its name. It's a little thing, but I prefer to vary my red herring in my test data to cover even more edge cases that way 😛

Reply:
> As for naming this `README.md`, since I already named the one in `Pods/SomePod/` as `README.md`, I wanted to use a different name here on purpose in order to ensure that any exclusion we'd do in our logic to ensure we only scan `.m` and `.swift` files would not just exclude the `README.md` common name but any `*.md` file whatever its name. 

Nice idea 👍